Automattic/jetpack
# JITM: move jetpack-jitm.js to the JITM package

#### Changes proposed in this Pull Request:

The JITM package should be able to be used independently of Jetpack. So, let's move the jetpack-jitm.js file out of Jetpack and into the JITM package.

* Update the monorepo:
    * Build the JITM package in the package.json `build_packages` and `build_production_packages` scripts.
    * Add `jetpack-jitm.js` to the eslint ignore list.
* Update the JITM package files:
    * Move the `jetpack-jitm.js` file to the JITM package.
    * Add a `webpack.config.packages.js` file.
    * Update the `.gitattributes` file.
    * Update the `.gitignore` file.
    * Add the `build-production` and `build-development` scripts to the composer.json file.
    * Add a `package.json` file.
    * Update the location of the jetpack-jitm.js file in `JITM::jitm_enqueue_files`.
    

#### Jetpack product discussion
* n/a

#### Does this pull request change what data or activity we track or use?
* no

#### Testing instructions:
1. Install and activate this branch and connect to wpcom.
2. Navigate to wp-admin -> Jetpack -> Dashboard. If you haven't already dismissed it, the `Give your site a speed boost` JITM should be displayed. Verify that it looks correct.

#### Proposed changelog entry for your changes:
* n/a - I don't think a changelog entry is needed for these changes.
